eZ/Publish/Core/MVC/Symfony/View/RelationView.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\MVC\Symfony\View;
+
+/**
+ * A view that contains relation data.
+ */
+interface RelationView
+{
+    /**
+     * Returns the Relations for the current entity being viewed.
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Relation[]
```

Comment:
Input from @bdunogier, instead of doing this no matter if relation is rendered or not we should use manual tagging in this case. there is a twig function by fos_http_cache for this.